### PR TITLE
Add Tree New Perks

### PR DESCRIPTION
**Pull request by Superfly-Johnson**

### CODE REVIEW

**Comment by pStyl3** [comment]
[TTW Reputations](https://www.nexusmods.com/newvegas/mods/68604/)

What's your reasoning for wanting to load `Tree New Perks.esp` after `TTW Reputations.esm`?

The **Tree New Perks** mod seems only to be available in the "Old Files" category of TTW - Wasteland Supplements, and it's description reads:
> Adds new unique perks for the various endings to the Oasis questline.
> 
> One of the perks could be broken, needs testing.

**Comment by Superfly-Johnson** [comment]
>What's your reasoning for wanting to load Tree New Perks.esp after TTW Reputations.esm?
Tree New Perks and TTW reputations conflict as they both modify the quest Oasis. Tree New Perks has an inline patch for TTW Reputations so it should be loaded afterwards

Tree new Perks used to have its own page but it waas abandonned and subsequently taken over by the wasteland supplements page from my understanding

File: masterlist.yaml [hunk 1]
```diff
@@ -2412,6 +2412,15 @@ plugins:
       - crc: 0x2EA38DC1
         util: 'FNVEdit v4.0.4c'
 
+  - name: 'Tree New Perks.esp'
+    url:
+      - link: 'https://www.nexusmods.com/newvegas/mods/79005/'
+        name: 'TTW - Wasteland Supplements'
+    after: [ 'TTW Reputations.esm' ]
```
Comment (MacSplody) [inline masterlist.yaml]: Tested and default sorting already places `Tree New Perks.esp` after `TTW Reputations.esm`.
Even when both files have an ESM flag.
```suggestion
```

**Comment by MacSplody** [comment]
Thanks.

**Pull request merged**